eng/UpdateAzureSdkForNet.ps1

```diff
@@ -13,4 +13,5 @@ $PackagesProps = "$SdkRepoRoot\eng\Packages.Data.props"
     "<PackageReference Update=`"Microsoft.Azure.AutoRest.CSharp`" Version=`"$Version`" PrivateAssets=`"All`" />" | `
     Set-Content $PackagesProps -NoNewline
 
+$Env:NODE_OPTIONS = "--max-old-space-size=8192"
```

Comment:
Just wondering - it looks like from the documentation for the hosted runners (https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml#hardware) the machines end up having 7GB of memory.  I have confirmed locally that the generation completes without error with `max-old-space-size=4096`.  Maybe we want to use that smaller value so we don't end up having to swap in CI?